ReactAndroid/src/main/java/com/facebook/react/views/slider/ReactSlider.java

```diff
@@ -1,36 +1,43 @@
 /**
  * Copyright (c) 2015-present, Facebook, Inc.
  * All rights reserved.
- *
+ * <p/>
  * This source code is licensed under the BSD-style license found in the
  * LICENSE file in the root directory of this source tree. An additional grant
  * of patent rights can be found in the PATENTS file in the same directory.
  */
 
 package com.facebook.react.views.slider;
 
+import android.annotation.TargetApi;
 import android.content.Context;
+import android.content.res.ColorStateList;
+import android.graphics.drawable.Drawable;
+import android.os.Build;
 import android.util.AttributeSet;
 import android.widget.SeekBar;
 
+import com.facebook.react.bridge.ReadableMap;
+
 import javax.annotation.Nullable;
 
 /**
  * Slider that behaves more like the iOS one, for consistency.
- *
+ * <p/>
  * On iOS, the value is 0..1. Android SeekBar only supports integer values.
  * For consistency, we pretend in JS that the value is 0..1 but set the
  * SeekBar value to 0..100.
- *
+ * <p/>
```

Comment:
In theory the `<p>` are good here for rendering HTML docs but don't think we use them across the codebase - can you check? It's more important to stay consistent.